dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/audit/AuditRepositoryTest.java

```diff
@@ -360,30 +359,10 @@ void testCompressDecompress()
     }
 
     @Test
-    @Disabled
-    void testAuditInsert200k()
+    void testAuditInsertBatch()
     {
         List<Audit> audits = new ArrayList<>();
-        IntStream.rangeClosed( 1, 200_000 ).forEach( n -> {
-            String uid = CodeGenerator.generateUid();
-            String code = CodeGenerator.generateUid();
-            Audit audit = Audit.builder().auditType( AuditType.CREATE ).auditScope( AuditScope.AGGREGATE )
-                .createdAt( LocalDateTime.of( 2000, 1, 1, 0, 0 ) ).createdBy( "test-user" )
-                .klass( DataElement.class.getName() ).uid( uid ).code( code ).data( "{}" ).build();
-            audits.add( audit );
-        } );
-        Timer timer = new Timer().start();
-        audits.forEach( audit -> auditRepository.save( audit ) );
-        System.err.println( "Single Insert: " + timer.getTimeInS() + "s" );
```

Comment:
Looks like these 2 tests are testing different things (just briefly looking at the assertion/println). Why did you decide to remove one?

Reply:
I removed the single insert 200K because it is already covered and it is more a performance test, it takes 4 minutes in my machine.
I enabled the batch one and decreased the number of insert in order to have an acceptable run time and still testing the save of multiple audits that was not covered